Apps/gladys/docker-compose.yml

```diff
@@ -11,50 +11,19 @@ services:
     image: gladysassistant/gladys:v4 # Docker image to use for the 'app' service
     restart: always # Container should always restart
     privileged: true # Grants additional privileges to this container
+    network_mode: host
+    container_name: gladys
+    cgroup: host
     environment: # Environment variables for the container
       NODE_ENV: production # Sets the environment to production
       SQLITE_FILE_PATH: /var/lib/gladysassistant/gladys-production.db # Path to the SQLite database
+      TZ: Europe/Paris # Timezone
+      SERVER_PORT: "1080" # Server port for UI
     volumes: # Mount points from the host to the container
       - /var/run/docker.sock:/var/run/docker.sock # Docker socket file for Docker within Docker
-      - /DATA/AppData/$AppID/data:/var/lib/gladysassistant # Persistent storage for Gladys data
```

Comment:
In CasaOS you put app data in /DATA/AppData/*

Reply:
Fixed